utils/frontend/pages/Swap.ts

```diff
@@ -30,6 +31,8 @@ export class Swap {
     buildDataTestIdXpath(DIV_SWAP_PAY) + "//*[contains(text(),'Max')]";
   private btnGetMaxLocator =
     buildDataTestIdXpath(DIV_SWAP_GET) + "//*[contains(text(),'Max')]";
+  private btnManageTokensList = buildDataTestIdXpath(BTN_MANAGE_TOKENS_LIST);
+  private btnToggleShowAllTokens = "//*[@for='tokenListsCheckbox-everything']";
```

Comment:
ask for data-test-ids to FE guys. 
it was agreed with them a long time ago to include those for automation porpuses.

Reply:
I expect selectors to change quite heavily with new trading app - so will guard and enforce these selectors there but not necessarily in current app version.